core/server/api/v2/integrations.js

```diff
@@ -1,9 +1,13 @@
-const Promise = require('bluebird');
 const models = require('../../models');
 
 module.exports = {
     docName: 'integrations',
-    browse: {},
+    browse: {
```

Comment:
Do we want to be able to get api_keys and webhooks when browsing?

Reply:
Yep, the browse endpoint is the only one Ghost-Admin will be hitting. One request to fetch everything should mean a much snappier UI and we don’t expect the amount of data to be huge